tests/std/include/test_is_sorted_until_support.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+
+#include <algorithm>
+#include <cassert>
+#include <cmath>
+#include <cstddef>
+#include <type_traits>
+#include <utility>
+#include <vector>
+
+#if _HAS_CXX20
+#include <ranges>
+#endif

```

Comment:
Perhaps surprisingly, `<algorithm>` is sufficient for `ranges::is_sorted_until`.